usr/src/man/man1m/zpool.1m

```diff
@@ -660,7 +661,7 @@ to the enabled state.
 See
 .Xr zpool-features 5
 for details on feature states.
-.It Sy listsnaps Ns = Ns Sy on Ns | Ns Sy off
+.It Sy listsnapshots Ns = Ns Sy on Ns | Ns Sy off
```

Comment:
Could please include the https://www.illumos.org/issues/3796 in the commit message for this change.